src/services/notifier/hooks/subscriptions.hook.ts

```diff
@@ -28,13 +28,22 @@ export default {
                 {
                   model: NotifierChannelModel,
                   as: 'channels',
-                  attributes: ['name'],
                   required: true
                 }
-              ]
+              ],
+              attributes: {
+                include: [
+                  [
+                    literal('(SELECT url FROM notifier_provider AS provider WHERE SubscriptionModel.providerId = provider.provider)'),
+                    'url'
+                  ]
+                ]
+              }
```

Comment:
this is unnecessary. just add
```
            {
              model: ProviderModel,
              as: 'provider',
              attributes: ['url']
            }
```
to the list of includes (sibling to the plan object).

Reply:
This is a nice suggestion. The problem is that this would nest the url prop under provider object. Which doesn't map with our SubscriptionDTO models in UI. So probably we need to update the models in UI as well but for now I'm resolving this conversation and closing this PR as we need to define first which data we need when fetching subscriptions.